src/wallet/components/WalletDropdownLink.tsx

```diff
@@ -35,7 +35,7 @@ export function WalletDropdownLink({
       target={target}
       rel={rel}
     >
-      <div className="-translate-y-1/2 absolute top-1/2 left-4 flex h-4 w-4 items-center justify-center">
+      <div className="-translate-y-1/2 absolute top-1/2 left-4 flex h-[1.125rem] w-[1.125rem] items-center justify-center">
```

Comment:
rem? are we sure this is stable? rem usually is relative to the screen.

Reply:
The previous setting (h-4 and w-4) are also using REM

Reply:
<img width="307" alt="Screenshot 2024-08-12 at 5 18 41 PM" src="https://github.com/user-attachments/assets/dae6bb8d-97db-4fdb-8ced-41eb95172a54">

Reply:
<img width="262" alt="Screenshot 2024-08-12 at 5 19 12 PM" src="https://github.com/user-attachments/assets/7fb6e397-89de-4169-8b7c-09ab493ed284">